cmake/kokkos_enable_options.cmake

```diff
@@ -48,6 +48,13 @@ KOKKOS_ENABLE_OPTION(COMPILER_WARNINGS    OFF "Whether to print all compiler war
 KOKKOS_ENABLE_OPTION(PROFILING_LOAD_PRINT OFF "Whether to print information about which profiling tools got loaded")
 KOKKOS_ENABLE_OPTION(TUNING               OFF "Whether to create bindings for tuning tools")
 KOKKOS_ENABLE_OPTION(AGGRESSIVE_VECTORIZATION OFF "Whether to aggressively vectorize loops")
+KOKKOS_ENABLE_OPTION(VIEW_HOOKS           OFF "Whether to enable View Hooks")
+
+IF(KOKKOS_ENABLE_VIEW_HOOKS)
+   IF(KOKKOS_CXX_STANDARD LESS 14)
```

Comment:
We can get rid of this now, yeah? 14 is the minimum?

Reply:
resolved with 66564ee